packages/scripts/src/migration/dvm2-designated-voting/0_Deploy.ts

```diff
@@ -0,0 +1,113 @@
+const hre = require("hardhat");
+const fetch = require("node-fetch");
+
+import {
+  VotingTokenEthers,
+  DesignatedVotingFactoryEthers,
+  DesignatedVotingEthers,
+  DesignatedVotingV2FactoryEthers,
+} from "@uma/contracts-node";
+import { getContractInstance } from "../../utils/contracts";
+
+const { getContractFactory } = hre.ethers;
+
+async function main() {
```

Comment:
Not necessary to add in this PR, but we may want to allow the caller to pass in remapped hot wallet addresses, as a few users would like their hot wallets changed.